QuickOSM/core/process.py

```diff
@@ -37,13 +37,14 @@
 __email__ = 'info@3liz.org'
 
 
+OSM_LAYERS = [LayerType.Points, LayerType.Lines, LayerType.Multilinestrings, LayerType.Multipolygons]
```

Comment:
I think you can move this to the definitions/osm.py file, it's used in other places